GETTING_STARTED.md

```diff
@@ -46,6 +46,17 @@ Add HammerJS to your application via [npm](https://www.npmjs.com/package/hammerj
 (such as the [Google CDN](https://developers.google.com/speed/libraries/#hammerjs)), or served 
 directly from your app.
 
+```bash
+npm install --save hammerjs @types/hammerjs
```

Comment:
I'd prefer to see `npm install --save hammerjs` and `npm install --save-dev @types/hammerjs`